sdk/how-to-guides/manage-storage.md

```diff
@@ -0,0 +1,232 @@
+> See first: [Juju | How to Manage Storage](https://juju.is/docs/juju/manage-storage)
+
+**Contents:**
+
+- [Implement storage management](#heading--implement-storage-management)
+- [Test storage management](#heading--test-storage-management)
+
+
+<a href="#heading--implement-storage-management"><h2 id="heading--implement-storage-management">Implement storage management</h2></a>
+
+- [Declare the storage in `charmcraft.yaml`](#heading--declare-the-storage-in-charmcraft-yaml)
+- [Observe the attached event and define an event handler](#heading--observe-the-attached-event-and-define-an-event-handler)
+
+
+<a href="#heading--declare-the-storage-in-charmcraft-yaml"><h3 id="heading--declare-the-storage-in-charmcraft-yaml">Declare the storage in `charmcraft.yaml`</h3></a>
+
+To define the storage that can be provided to the charm, define a `storage` section in `charmcraft.yaml` that lists the storages and information about each storage. For example, for a transient filesystem storage mounted to `/cache/` that is at least 1GB in size:
```

Comment:
Storage volumes?

Reply:
Looking over the existing docs, we seem to have a mix of "storage volumes", "storage instances", and "storages", I believe all meaning the same thing.

I agree "storages" is probably not the best choice. Do you have a preference between "storage volumes" and "storage instances"? "volume" has fairly specific meaning in filesystems and in K8s, so probably "storage volumes" is clearest?

Reply:
> so probably "storage volumes" is clearest?

Yes.